### ISSUE TITLE
Designer Shadow DOM

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-core#2976]

**Comment by tomy2105** [comment]
Encapsulation, from programming perspective is always nice.
However, I kind of liked the fact that you can "style" Elsa elements using global CSS (althought I faces troubles when Elsa CSS also styled my own ones :)).
When using shadow dom how will you enable "end" application to style Elsa components?

**Comment by sfmskywalker** [comment]
Yeah good point - we should still be able to customize the look & feel by CSS-targeting the shadow DOM. Which should be possible with the `:shadow` pseudo element and/or shadow parts.

https://www.html5rocks.com/en/tutorials/webcomponents/shadowdom-201/#toc-style-cat-hat
https://css-tricks.com/styling-in-the-shadow-dom-with-css-shadow-parts/
